internal/gcsx/bucket_manager_test.go

```diff
@@ -42,6 +42,30 @@ func (t *BucketManagerTest) TearDown() {
 	t.fakeStorage.ShutDown()
 }
 
+func (t *BucketManagerTest) TestCreateStatCacheLru() {
+	inputs := []struct {
+		capacity int
+		isNil    bool
+	}{
+		{
+			capacity: 0,
+			isNil:    true,
+		},
+		{
+			capacity: -1,
+			isNil:    false,
+		},
+		{
+			capacity: 10,
+			isNil:    false,
+		},
+	}
+
+	for _, input := range inputs {
```

Comment:
@gargnitingoogle @sethiay Can someone help me understand a scenario... will this test pass if we add capacity as 10 and output from `createStatCacheLru` is returned 5 (hypothetically if there is bug in the method), if yes, then will it be good idea to test exact output values in three separate asserts or is this the way we should do in GoLang?

Reply:
createStatCacheLru return a lru.Cache instance given the size. 
If size is <=0, it should return nil. 
For size>0, a non-nil object should be returned. But there is no easy way to verify that the return object has the passed-in size as it has no interface for that. So, we're testing only for whether the object got created.
More detailed tests for statCache are in https://github.com/googlecloudplatform/gcsfuse/blob/5c8ef1cd1ed4572033b6c47e5297141d3f2bd64c/internal/cache/metadata/stat_cache_test.go .